samples/nrf5340/netboot/README.rst

```diff
@@ -1,88 +1,92 @@
 .. _nc_bootloader:
 
-nRF5340: Network core bootloader
+nRF5340: Network Core Bootloader
 ################################
 
 .. contents::
    :local:
    :depth: 2
 
-This bootloader sample implements an immutable first stage bootloader that has the capability to update the application firmware on the network core of the nRF5340 System on Chip (SoC).
-In addition to this, the network core bootloader sample performs flash protection of both itself and the application.
+The Network Core Bootloader sample, also called B0n, is an immutable first-stage bootloader that can update the application firmware on the network core of the nRF5340 System on Chip (SoC).
 
 Overview
 ********
 
-The network core bootloader sample supports transporting an already verified and authenticated firmware upgrade from the application core flash to the network core flash, as well as performing flash protection.
+The Network Core Bootloader sample protects the flash memory areas allocated to both itself and the application running on the network core.
 
-This is accomplished by the following steps:
+You must use this sample as a child image of a :ref:`multi-image <ug_multi_image>` build, where MCUboot is enabled and there is a network core application.
+MCUboot verifies and shares with the network core bootloader any new network core application image received through a device firmware update (DFU) transport layer, like a serial or a Bluetooth LE connection.
+For this reason, without MCUboot, this sample does nothing other than launching directly the application.
 
-1. Lock the flash of the bootloader.
-     The bootloader sample locks the flash that contains the sample bootloader and its configuration.
+During the boot process, the Network Core Bootloader sample and MCUboot interact as follows:
+
+1. MCUboot writes the Network Core Bootloader sample configuration to the shared SRAM.
```

Comment:
MCUboot only writes a _command_ to the Network Core Bootloader if it identifies a network core application image. MCUboot does not write anything to shared SRAM if no network core update is found.

Reply:
fixed